### ISSUE TITLE
Installation typos 

### ISSUE THREAD

**Issue by wittend** [mloebl/mqtt-aprs#1]
Thanks for this!

A few typos complicate installation as described in the README.md file:

Line:
    git clone git://github.com/mloebl/mqtt-mqtt-aprs.git /usr/local/mqtt-aprs/
should read:
    git clone git://github.com/mloebl/mqtt-aprs.git /usr/local/mqtt-aprs/

line:
    pdate-rc.d mqtt-aprs defaults
should read:
    update-rc.d mqtt-aprs defaults

line:
    cp /usr/local/mqtt-aprsp/mqtt-aprs.default /etc/default/mqtt-aprs
should read:
    cp /usr/local/mqtt-aprs/mqtt-aprs.default /etc/default/mqtt-aprs

On my Ubuntu 18.04 Intel system the following also occured:
    dave@dillo:~/Projects/python-dev$ sudo /etc/init.d/mqtt-aprs start
    sudo: /etc/init.d/mqtt-aprs: command not found

This required something to the effect:
    sudo chmod a+x /etc/init.d/mqtt-aprs
before it would work.

**Comment by EmilyNerdGirl** [comment]
Thanks, I'll take a look!  I did this as an experiment, glad someone tried it out :)

**Comment by wittend** [comment]
I'm interested in this and related topics generally.  I might like to know
more about your thoughts generally.

Dave Witten KD0EAG

**Comment by EmilyNerdGirl** [comment]
A few years ago I was looking to bridge some 1wire devices to MQTT, and found this:
https://github.com/kylegordon/mqtt-owfs-temp

It hadn't been maintained for a bit, and I needed to add some additional devices, I forked it and made a few updates, switched to paho mqtt, and added a few add'l options. I then use it as a cookie-cutter template when playing with MQTT services (hence the embarrassing typos). 

A lot of my MQTT work centers around usage at home with home automation.

I don't recall what drove me to attempting this; I think it was to see if I could do it, and maybe use it at some point for tracking purposes.  I used to maintain an APRSParser plugin a few years back for some home automation software, so took up the challenge to see if I could do something similiar with MQTT.
